sdk/communication/communication-phone-numbers/test/public/lro.update.spec.ts

```diff
@@ -17,7 +17,9 @@ matrix([[true, false]], async function (useAad) {
 
     before(function (this: Context) {
       const skipPhoneNumbersTests = env.COMMUNICATION_SKIP_INT_PHONENUMBERS_TESTS === "true";
-      if (skipPhoneNumbersTests) {
+      const includePhoneNumberLiveTests = env.INCLUDE_PHONENUMBER_LIVE_TESTS === "true";
```

Comment:
Could you rewrite this so that you do not have so many ! in your boolean expression below?

Reply:
@whisper6284 thanks for the feedback. I just sent out a commit addressing it, could you please review?